packages/core/src/controllers/verify.ts

```diff
@@ -35,9 +35,10 @@ export class Verify extends IVerify {
     }
 
     if (!this.iframe) return;
-
-    this.iframe.contentWindow?.postMessage(params.attestationId, this.verifyUrl);
-    this.logger.info(`postMessage sent: ${params.attestationId} ${this.verifyUrl}`);
+    try {
+      this.iframe.contentWindow?.postMessage(params.attestationId, this.verifyUrl);
+      this.logger.info(`postMessage sent: ${params.attestationId} ${this.verifyUrl}`);
+    } catch (e) {} // fail silently to avoid logging `Failed to execute 'postMessage' on 'DOMWindow': The target origin provided...` while the iframe is still loading
```

Comment:
Are there any other "actionable" errors we should still be throwing?

Reply:
not that I'm aware of. Additionally, the verify is not functional at the and will be re-released after the migration